packages/lit-dev-content/site/articles/article/lit-for-polymer-users.md

```diff
@@ -0,0 +1,1021 @@
+---
+title: Lit for Polymer users
+publishDate: 2022-08-23
```

Comment:
The distant future.... The year 2000!

Reply:
Oops. More importantly, perhaps, my updated date is in the distant _past_. Heh.